share/pot/common-flv.sh

```diff
@@ -102,9 +103,26 @@ _exec_flv()
 					if ! _flv_set_cmd "$_pname" "$line" ; then
 						return 1
 					fi
+				elif [ "$line" != "${line#copy-in-flv}" ]; then
+					# copy-in relative to flavour dir
+					_previous_pwd=$PWD
+					if ! cd "$_flv_dir"; then
+						_error "Can't chdir to flavour dir $_flv_dir"
+						return 1
+					fi
+					line=$(echo "$line" | sed "s/^copy-in-flv/copy-in/")
+					if ! pot-cmd $line -p "$_pname" ; then
+						return 1
+					fi
+					if ! cd "$_previous_pwd"; then
+						_error "Can't chdir to previous pwd $_previous_pwd"
+						return 1
+					fi
 				else
 					# shellcheck disable=SC2086
-					pot-cmd $line -p "$_pname"
+					if ! pot-cmd $line -p "$_pname" ; then
```

Comment:
This shellcheck seems unnecessary (even though I don't really get why). Shellcheck isn't complaining though (that's why I don't have it above)